README.md

```diff
@@ -9,70 +9,16 @@ The authors take no responsibility for any loss of digital assets or other damag
-// Thread.sleep(2000);
-// This seems to be the correct path for persistent files.
-String appDir = getApplicationContext().getFilesDir().getAbsolutePath();
-String dbPath = appDir +"/database";
-// 10.0.2.2 is the IP of the host PC when using Android Simulator and the host is running a ganache-cli.
-// 8545 is the standart port of ganache-cli.
-String ethUrl = "ws://10.0.2.2:8545";
-// Create a wallet.
-Wallet wallet = new Wallet(appDir +"/wallet", "password");
-// We are alice in this example and this is our on-chain secret key holding the ETH.
-Address alice = wallet.importAccount("0x69cb97043e56883d66627e8f7a828877a56022d0fb05ae6197e6e16fb56282d0");
-// Using null as either Adjudicator or Assetholder tells the Client to deploy the contracts,
-// in this case we already deployed them.
-Address adjudicator = new Address("0xDc4A7e107aD6dBDA1870df34d70B51796BBd1335");
-Address assetholder = new Address("0xb051EAD0C6CC2f568166F8fEC4f07511B88678bA");
-// Listen on 127.0.0.1:5750 for channel Proposals.
-Config cfg = new Config("Alice ", alice, adjudicator, assetholder, dbPath, ethUrl, "127.0.0.1", 5750);
-// Create a client, currently skipping the ProposalHandler.
-Client client = new Client(cfg, wallet);
-// PerunId (currently an Address) of the peer that we want to open a channel with.
-Address bob = new Address("0xA298Fc05bccff341f340a11FffA30567a00e651f");
-// Tell the client what `bob`s ip and port are.
-client.addPeer(bob, "10.0.2.2", 5750);
-// Create the initial balances of the channel, me starting with 2000 and bob with 1000.
-BigInts initBals = Prnm.newBalances(new BigInt(2000), new BigInt(1000));
-// The ongoing channel proposal can be cancelled with ctx.cancel().
-Context ctx = Prnm.contextWithCancel();
-// Propose a channel to `bob` with `initBals` and 60s challenge duration.
-PaymentChannel channel = client.proposeChannel(ctx, bob, 60, initBals);
-// ctx.cancel() must always be called or it will leak resources.
-// Better put it in the `finally` block of the surounding `try`.
-ctx.cancel();
-```  
-This code must be wrapped in a `try` as the compiler will let you know.  
-You can always `Ctrl+Click` on a *prnm* function to use the Android Studio decompiler, wich is really helpful to see all available Java functions.
-
-The implementation of incoming channel proposals and channel updates will be added shortly.  
-
-
-### Android App permissions
-The `AndroidManifest.xml` file must contain at least the following permissions for the app.  
-```xml
-<uses-permission android:name="android.permission.INTERNET" />
-<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-```
-Place this above the `<application>` section.
+Import the `android/` folder as Project into Android Studio and run it.
```

Comment:
We can definitely shorten the README but still, add a few sentences explaining which are the most important files in the `android/` example project. In particular, explain very high-level what is happening in file `android/app/src/main/java/network/perun/app/MainActivity.java` and that the permissions from file `android/app/src/main/AndroidManifest.xml` are relevant.

Reply:
I changed it please have a look.

Reply:
looks good now, thanks